apps/judicial-system/backend/src/app/modules/case/test/internalCaseController/deliverIndictmentAssignedRolesToCourtGuards.spec.ts

```diff
@@ -0,0 +1,50 @@
+import { CanActivate } from '@nestjs/common'
+
+import { indictmentCases } from '@island.is/judicial-system/types'
+
+import { CaseExistsGuard } from '../../guards/caseExists.guard'
+import { CaseTypeGuard } from '../../guards/caseType.guard'
+import { InternalCaseController } from '../../internalCase.controller'
+
+describe('InternalCaseController - Deliver assigned roles in indictment case to court guards', () => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  let guards: any[]
+
+  beforeEach(() => {
```

Comment:
I have started using a different pattern in guard tests. See for instance `apps/judicial-system/backend/src/app/modules/case/test/caseController/getByIdGuards.spec.ts`.

Please consider using the new pattern as it runs faster.